src/libstd/sys/redox/ext/io.rs

```diff
@@ -109,6 +110,21 @@ impl AsRawFd for net::UdpSocket {
     }
 }
 
+#[stable(feature = "rust1", since = "1.0.0")]
```

Comment:
What should `feature` be? I don't know exactly how this works, since trait implementations aren't behind feature flags.